create_derivatives/routines.py

```diff
@@ -326,7 +329,7 @@ def create_manifest(self, identifier, obj_data):
             canvas.thumbnail = self.set_thumbnail(filename)
             page_number += 1
         v2_json = manifest.toJSON(top=True)
-        if self.api_version == 2:
+        if self.presentation_api_version == 2:
             manifest_json = v2_json
         else:
             v3_json = self.upgrader.process_resource(v2_json, top=True)
```

Comment:
```suggestion
            manifest_json = self.upgrader.process_resource(v2_json, top=True)
```
And then delete following line.